cypress/integration/seededFlows/searchFlows/searchForm.spec.js

```diff
@@ -0,0 +1,10 @@
+describe('Search form', () => {
+  it('should show results when search button is pressed', () => {
+    cy.visit('/');
+
+    cy.findByRole('textbox', { name: /search/i }).type('test');
+    cy.findByRole('button', { name: /search/i }).click();
+
+    cy.url().should('include', '/search?q=test');
```

Comment:
```suggestion
    cy.url().should('include', '/search?q=test');
    cy.findByRole('heading', { name: 'Test article' }).should('exist');
```

Just suggesting a small addition here that will make sure the list of search results has populated, as well as the UR: having changed in the browser.

Reply:
I'm going to go ahead and merge this suggestion from @aitchiss.